src/styles/blocks/_logo.scss

```diff
@@ -0,0 +1,7 @@
+.logo {
```

Comment:
Do we really need this block ? (just a question, you don't need to delete this)

Reply:
I think it might be a reusable block later, for instance in the footer, so I put it in its own file.